odoo/tools/query.py

```diff
@@ -203,7 +203,7 @@ def subselect(self, *args: str | SQL) -> SQL:
                 # in case we have nothing, we want to use a sub_query with no records
                 # because an empty tuple leads to a syntax error
                 # and a tuple containing just None creates issues for `NOT IN`
-                return SQL("(SELECT 1 WHERE FALSE)")
```

Comment:
Is it necessary ? I don't get it.

Reply:
I will roll it back and test why I needed it. I could probably replace it by a call to `is_empty`.